### PR TITLE
Twitters-color

### PR DESCRIPTION
**Pull request by KarlGusta**
## Changes proposed
Changing the twitter color to the original one.

Also, just practicing PR here TBH. 

## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1551"></a>

### CODE REVIEW

File: src/config/links.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "telegram": "#4995be",
     "tiktok": "#000000",
     "twitch": "#9146FF",
-    "twitter": "#00ACEE",
+    "twitter": "#1D9BF0",
```
Comment (kumarsonsoff3) [inline src/config/links.json:35]: The official color for Twitter is `rgb(29, 155, 240)` which is `#1d9bf0`. It should be 👇 
```suggestion
    "twitter": "#1d9bf0",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your PR should only contain the file in which you are making changes! So revert the changes from the `GetIcons.js`. 
As per my knowledge we've already merged your another PR sorting the icons! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Comment by kumarsonsoff3** [comment]
Hey @KarlGusta, I see your [fork](https://github.com/KarlGusta/LinkFree/tree/twitters-color) is behind the main branch, so this PR shows 2 files changed! 

Please Sync you fork with the linkfree main! let us know if you need any help with this!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by SaptarshiSarkar12** [comment]
@KarlGusta Please resolve the branch conflicts.

**Comment by KarlGusta** [comment]
Thank you for your patience @kumarsonsoff3 and @SaptarshiSarkar12 . I'm only seeing this now. It appears I wasn't getting any notification on this until today. Let me do a sync then do another PR. I'll close this for now.

**Pull request closed without merging**